class-events-manager-securesubmit.php

```diff
@@ -525,48 +534,40 @@ function authorize_and_capture($EM_Booking){
         global $EM_Notices;
 
         if( !class_exists('HpsServicesConfig') ){
```

Comment:
We need to update this logic as `HpsServicesConfig` isn't defined in the new SDK.

Reply:
updated